### ISSUE TITLE
Unable to sort stacks by server/agent

### ISSUE THREAD

**Issue by miltuss** [louislam/dockge#481]
### ⚠️ Please verify that this bug has NOT been reported before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### Description

I would like to be able to sort the stacks by server, but no filter exists in Dockge allowing me to sort this!  While it is essential to separate the stacks of each server for logical reasons of findability.  The name indicated below each stack is not enough to perform the sort.

### 👟 Reproduction steps

---

### 👀 Expected behavior

---

### 😓 Actual Behavior

---

### Dockge Version

1.4.2

### 💻 Operating System and Arch

debian 12

### 🌐 Browser

firefox

**Comment by ESPGranEdu** [comment]
Yeah, I also like the idea to sort the stacks per server since I have quite a few stacks across multiple servers hehe.
I worked on this nearly a month ago -> #554

I don't know if you're refering to this, but it could help :D

**Comment by olvier** [comment]
I'm new to dockge (today is day zero :)), but i'm loving it so much :)
Moving from portainer to dockge now <3

Would really really love to see a filter by tags, too.
You have done great with uptime-kuma :)

Maybe there is some option to implement some filter for the multiple agents?
Would really love <3



<img width="475" alt="Image" src="https://github.com/user-attachments/assets/7a5d46e4-d223-4959-a42b-80aa4816ebb6" />

**Comment by BriseBolt** [comment]
Why does the Dockge developer refuse to integrate this rather simple feature?

This feature is simple and yet so useful. While Uptime Kuma already has filters integrated into it, it's incomprehensible!

**Comment by EricZimmerman** [comment]
> Why does the Dockge developer refuse to integrate this rather simple feature?
> 
> This feature is simple and yet so useful. While Uptime Kuma already has filters integrated into it, it's incomprehensible!

maybe you should do a PR?

**Comment by GalacticCheat** [comment]
I've been waiting for this feature for a long time too!

@louislam When will you finally allow dockge users to filter stacks by agents/servers?

**Comment by DrakeDrole** [comment]
Does anyone know why @louislam abandoned dockge?
